### DISCUSSION TITLE
What is the selection criteria for npm versions while releasing NodeJS

### DISCUSSION THREAD

**Discussion by yeikel** [nodejs/help#4771] [Q&A] [answered]
Hello there, 

First of all, sorry if this has been documented. I could not find any relevant document (or I did not check in the right places) 

According to the downloads site, the latest build of Node 18(18.17.1 - 2023-08-08 ) shipped with NPM 9.6.7 and we had higher versions of NPM available at the time. Is it just a matter of timings?

Currently, I am tracking them separately (within the semantic spec) and I'd like to know what is best

Any inputs would be appreciated

**Comment by richardlau** [comment]
For an LTS release line (e.g. Node.js 18), we have a default policy that for something to be released it in that it should have previously been in a "Current" (e.g. at this point in time a Node.js 20 release) for two weeks. This policy is in place to allow unforeseen regressions to surface in the current release in an effort to maximize the stability of the LTS release lines.

Refs: https://github.com/nodejs/release#lts-staging-branches

**Reply by yeikel** [reply]
> This policy is in place to allow unforeseen regressions to surface in the current release in an effort to maximize the stability of the LTS release lines.

Would you recommended that users stick to the npm versions released with NodeJS?

**Reply by richardlau** [reply, marked as answer]
In general, yes.
